<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizarPedido</class>
 <widget class="QWidget" name="VisualizarPedido">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualizar Pedido</string>
  </property>
  <widget class="QTableWidget" name="tblItem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>941</width>
     <height>192</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>192</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>156</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Produto</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descrição</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor do Produto</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fornecedor</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantidade</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="notaFiscalLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nota Fiscal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTotal">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>310</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Total:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>280</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Desconto:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSair">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>310</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor Frete:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Transportadora:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbNota">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbTransportadora">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbData">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbFrete">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbDesconto">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>280</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
